let b = 2;

try{
    a = 2 + b;
}
catch(e){
    console.log(e);
}
finally{
    console.log('Executado');
}
